Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -8998,7 +8998,9 @@ public void HandleActUIElement(ActUIElement act)
                     case ActUIElement.eElementAction.ScrollToElement:
                         try
                         {
-                            ((IJavaScriptExecutor)Driver).ExecuteScript("arguments[0].scrollIntoView(true);", e);
+                            string scrollAlignment = act.GetInputParamCalculatedValue(ActUIElement.Fields.ScrollAlignment).ToLower();
+                            string command = $"arguments[0].scrollIntoView({{ block: '{scrollAlignment}' }});";
```

Comment:
According to Mozilla documentation, the scrollIntoView(scrollIntoViewOptions) feature is experimental. Execute the previous script when the value is set to default, and use the new script with viewOptions for any non-default value.

Reply:
Good catch. I decided to refactor the logic to avoid using `scrollIntoView(options)` and just use `scrollTo()` where necessary instead.